cmd/release/main.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+)
+
+/*
+required parameteres
+
+https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line
+github token for modifying release
+
+# dependencies
+* requires a build environment (make artifact-release must work)
+* github.com/itchio/gothub
```

Comment:
Should this info also be added to the release doc?

Reply:
Yep, if this type of automated release looks good for now I can go ahead and add the documentation for releasing